src/types.ts

```diff
@@ -3,6 +3,6 @@ export type EventContents = Record<string, any>
 export interface PubsubMessage {
     timestamp: string
     eventName: string
-    correlationId: string
+    correlationId?: string
```

Comment:
Husk å endre dette i typen i entur-bff også da.

Reply:
Eventuelt kan eg generere UUID her for å alltid vere sikker på at det er correlation ID. Er det bedre?

Reply:
Eg la til generering av correlation ID med UUID dersom det ikkje skulle eksistere i headers.